packages/expo-firebase-analytics/src/FirebaseAnalyticsJS.ts

```diff
@@ -172,6 +172,19 @@ class FirebaseAnalyticsJS {
     }
   }
 
+  private static isValidName(name: string, maxLength: number): boolean {
+    return !!(
+      name &&
+      name.length &&
+      name.length <= maxLength &&
+      name.match(/^[A-Za-z][A-Za-z_\d]*$/) &&
+      name !== 'user_id' &&
```

Comment:
I think this only applies to the userPropertyName case and not the event-name case (name !== 'user_id')

Reply:
Ah. Let me add this to the unit tests as well. 

<a href="https://gitme.me/image?url=https%3A%2F%2Fmedia3.giphy.com%2Fmedia%2FADr35Z4TvATIc%2Fgiphy.gif&token=faceplam" data-gitmeme-token="faceplam"><img src="https://media3.giphy.com/media/ADr35Z4TvATIc/giphy.gif" title="Created by gitme.me with /faceplam"/></a>

Reply:
Haha, sorry for being so nit-picky man 😅